src/context_store/lifecycle/manager.py

```
@@ -0,0 +1,850 @@
+                purger_result.purged_count,
+                purger_result.checked_count,
+            )
+
+            # 5. Stats Collector
+            await self._collect_stats()
+
+            # 6. WAL チェックポイント（SQLite のみ）
+            if self._wal_checkpoint_fn is not None:
+                await self._run_wal_checkpoint()
+
+            # 状態を更新（カウンターリセット + last_cleanup_at 更新）
```

Ruff (0.15.7)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)